src/Sql/Sql.Test/ScenarioTests/DataSyncTests.cs

```diff
@@ -20,121 +20,116 @@
 
 namespace Microsoft.Azure.Commands.Sql.Test.ScenarioTests
 {
-    public class DataSyncTests : SqlTestsBase
+    public class DataSyncTests : SqlTestRunner
     {
         public DataSyncTests(ITestOutputHelper output) : base(output)
         {
-            XunitTracingInterceptor.AddToContext(new XunitTracingInterceptor(output));
 
-            base.resourceTypesToIgnoreApiVersion = new string[] {
-                "Microsoft.Sql/servers",
-                "Microsoft.Sql/managedInstances/databases"
-            };
         }
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestSyncAgentCreate()
         {
-            RunPowerShellTest("Test-CreateSyncAgent");
+            TestRunner.RunTestScript("Test-CreateSyncAgent");
```

Comment:
Is it possible to add method RunPowerShellTest in super class `SqlTestRunner`. Then you don't need to change below each line.

Reply:
Wrap the TestRunner.RunTestScript with the RunPowerShellTest method is not that good because it is just a wrapper without any other function like concealing the real API. If some of the test cases need to pass in additional parameters later on, the wrapper won't work. In addition, adding a wrapper cannot reduce the size of the PR as all these classes must inherit from SqlTestRunner. This is a required change.